Circuit/Components/Tubes.xml

```diff
@@ -1,24 +1,39 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Library>
-	<!-- http://www.vcu.edu/csbc/bnfo601/Paul_amp_designs/Koren_Names.inc -->
-	<Component _Type="Circuit.Triode, Circuit, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"
+  <!-- http://www.vcu.edu/csbc/bnfo601/Paul_amp_designs/Koren_Names.inc -->
+  <Component _Type="Circuit.Triode, Circuit, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"
 			   Mu="67.49" K="1.73E-06" Name="V1" PartNumber="12AT7"
 			   Ex="1.234" Kg="419.1" Kp="213.96" Kvb="300" Rgk="1 MΩ" Vg="330 mV" />
-	<Component _Type="Circuit.Triode, Circuit, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"
+  <Component _Type="Circuit.Triode, Circuit, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"
 			   Mu="21.5" K="1.73E-06" Name="V1" PartNumber="12AU7"
 			   Ex="1.3" Kg="1180" Kp="84" Kvb="300" Rgk="1 MΩ" Vg="330 mV" />
-	<Component _Type="Circuit.Triode, Circuit, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"
+  <Component _Type="Circuit.Triode, Circuit, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"
 			   Mu="83.5" K="1.73E-06" Model="DempwolfZolzer" Name="V1" PartNumber="12AX7"
 			   Ex="1.4" Kg="1060" Kp="600" Kvb="300" Rgk="1 MΩ" Vg="330 mV"
 			   Gamma="1.26" G="2.242e-3" Gg="6.177e-4" C="3.4" Cg="9.901" Xi="1.314" Ig0="8.025e-8 A" />
-	<Component _Type="Circuit.Triode, Circuit, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"
+  <Component _Type="Circuit.Triode, Circuit, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"
 			   Mu="44.16" K="1.73E-06" Name="V1" PartNumber="12AY7"
 			   Ex="1.113" Kg="1192.4" Kp="409.96" Kvb="300" Rgk="1 MΩ" Vg="330 mV" />
 	<Component _Type="Circuit.Triode, Circuit, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"
 			   Mu="74.08" K="1.73E-06" Name="V1" PartNumber="12AZ7"
 			   Ex="1.371" Kg="382" Kp="190.11" Kvb="300" Rgk="1 MΩ" Vg="330 mV" />
-	<!-- http://www.frank.mif.pg.gda.pl/other/docs/Koren_Tubes.cir -->
-	<Component _Type="Circuit.Triode, Circuit, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"
```

Comment:
Looks like there is a git issue, this tube should not be removed.

Reply:
Something went wrong during merge, thanks for catching this! 👍